### PR TITLE
Update docs around installing packages via git*

### PR DESCRIPTION
**Pull request by djgoku**
### Contributor checklist

- [X] Features include unit/acceptance tests

***One thing I dislike is the test for the `package` module, as it involves making a web request.***

This adds more documentation around this work https://github.com/ash-project/igniter/commit/05422953e00bee555bbc6ac8eda86650b4592639 and this issue #84

### CODE REVIEW

**Comment by djgoku** [comment]
Closing my PR since this was updated also via https://github.com/ash-project/igniter/pull/157

File: lib/igniter/project/deps.ex [hunk 1]
```diff
@@ -312,7 +312,8 @@ defmodule Igniter.Project.Deps do
 
           "github:" <> requirement ->
             if String.contains?(requirement, "@") do
-              case String.split(requirement, ["/", "@"], trim: true) do
+              # adding parts will allow for branch names with forward slashes
+              case String.split(requirement, ["/", "@"], trim: true, parts: 3) do
```
Comment (zachallaun) [inline lib/igniter/project/deps.ex:316]: Good catch! I'll make sure this is added and tested in the other PR.

**Comment by zachallaun** [comment]
Thanks for this! I'll make sure the various improvements here are reflected in the other PR as well :)

**Comment by djgoku** [comment]
> Thanks for this! I'll make sure the various improvements here are reflected in the other PR as well :)

I appreciate the review and my test addition. I intended to leave a note if you thought it was worth adding.

**Pull request closed without merging**